object_database/web/cells/util.py

```diff
@@ -312,3 +312,23 @@ def MarginSides(amount, cell=None):
         return CellDecorator(lambda cell: MarginSides(amount, cell), f"MarginSides({amount})")
 
     return CustomInset(cell, "margin", top=0, right=amount, bottom=0, left=amount)
+
+
+def BackgroundColor(color, cell=None):
+    """Cell modifier function that
+    will set the customStyle background-color
+    for a given Cell instance.
+
+    Parameters
+    ----------
+    color: str
+        A CSS-compatible string value for
+        a background color
+
```

Comment:
It was unclear to me (because I know so little about CSS) whether only color names were supported, but I see "#ff00ff" strings are also supported. Maybe, add an example with a color name and one with an RGB color string

Reply:
Would you like me to add these to the CellsTestPage examples or in the comments?